### PR TITLE
Screenshots fixed

### PR DESCRIPTION
**Pull request by Bartixxx32**

### CODE REVIEW

File: admin_client.lua [hunk 1]
```diff
@@ -115,9 +115,9 @@ end)
 
 
 AddEventHandler('EasyAdmin:CaptureScreenshot', function(toggle, url, field)
-	exports['screenshot-basic']:requestScreenshotUpload(GetConvar("ea_screenshoturl", 'https://wew.wtf/upload.php'), GetConvar("ea_screenshotfield", 'files[]'), json.decode(GetConvar("ea_screenshotOptions", "{}")) , function(data)
+	exports['screenshot-basic']:requestScreenshotUpload(GetConvar("ea_screenshoturl", 'https://wew.wtf/upload.php'), GetConvar("ea_screenshotfield", 'files[]'), function(data)
 			local resp = json.decode(data)
```
Comment (Bluethefurry) [inline admin_client.lua]: you can remove this line if its unused anyway

File: admin_server.lua [hunk 2]
```diff
@@ -435,16 +435,12 @@ Citizen.CreateThread(function()
...
-				if not json.encode(result) or not json.encode(result).files or not json.encode(result).files[1] or not json.encode(result).files[1].url then
-					response = json.encode(result)[1].url
-				else
-					response = result
-				end
-				TriggerClientEvent('chat:addMessage', src, { template = '<img src="{0}" style="max-width: 400px;" />', args = { response } })
-				TriggerClientEvent("chat:addMessage", src, { args = { "EasyAdmin", string.format(GetLocalisedText("screenshotlink"), response) } })
-				PrintDebugMessage("Screenshot for Player "..getName(playerId,true).." done, "..response.." requsted by"..getName(src,true))
-				SendWebhookMessage(moderationNotification, string.format(GetLocalisedText("admintookscreenshot"), getName(src), getName(playerId), response))
+				response2 = result
```
Comment (Bluethefurry) [inline admin_server.lua]: why this unused convar?

File: admin_server.lua [hunk 3]
```diff
@@ -454,7 +450,7 @@ Citizen.CreateThread(function()
 			repeat
 				timeoutwait=timeoutwait+1
 				Wait(5000)
-				if timeoutwait == 5 then
+				if timeoutwait == 2 then
```
Comment (Bluethefurry) [inline admin_server.lua]: ?? why?

Comment (Bartixxx32) [reply admin_server.lua]: long time to next screenshot ( will revert

File: admin_server.lua (hunk 2 above)
Comment (Bartixxx32) [reply admin_server.lua]: response2 = result ?

File: admin_client.lua (hunk 1 above)
Comment (Bartixxx32) [reply admin_client.lua]: local resp = json.decode(data) ?

**Pull request merged**